data/fonts/OverlayFont.json

```diff
@@ -4,14 +4,14 @@
    "faces" : [
       {
          "fontFile" : "PionilliumText22L-Medium.ttf",
-         "pixelWidth" : 14,
```

Comment:
Is this change meant to influence imgui? font sizes are still hardcoded for imgui, these settings don't do anything for it. If it is meant to work for C++, that's correct then ;)

Reply:
This changes the fonts that are rendered in the 3D portion of the sector map, like the system labels. It also changes a few others, like the auto-pilot/comms window.
I was originally going to do this as a separate commit, but @nozmajner felt it impacted the sector map enough to include it here.

Reply:
> but @nozmajner felt it impacted the sector map enough to include it here.

Hey-hey, it's spelled "impakted" please. 😃

Reply:
Yup, it was my suggestion, to get the text sizes of the map labels closer to the imgui parts. I think it looks way better this way.